render/container.go

```diff
@@ -264,6 +264,11 @@ func MapContainer2IP(m report.Node) []string {
 			if !ok {
 				continue
 			}
+			// loopback addresses are shared among all namespaces
+			// so we can't use them to attribute connections to a container
+			if report.IsLoopback(addr) {
+				continue
+			}
```

Comment:
won't this exclude a container connecting to itself, due to the scoped loopback interface?

Also, `host1;127.0.0.1;8080` maps to a specific pid, right? because it has the host, ip, and port?